docs/Parameters.rst

```diff
@@ -1275,8 +1275,7 @@ LightGBM supports weighted training. It uses an additional file to store weight
     0.8
     ...
 
-It means the weight of the first data row is ``1.0``, second is ``0.5``, and so on. Weight should be non-negative
-values because the Hessian value multiplied by weight is supposed to be non-negative.
+It means the weight of the first data row is ``1.0``, second is ``0.5``, and so on. Weights should be non-negative.
```

Comment:
Thank you for the contribution. You may use the script
https://github.com/microsoft/LightGBM/blob/master/helpers/parameter_generator.py
to generate Parameters.rst after modifying the description of parameters in config.h. In this way, the documentation will be consistent with the description in the code, and the static checks won't fail due to inconsistency.

Reply:
thanks, it would be difficult to figure that on my own